### ISSUE TITLE
Bottom text

### ISSUE THREAD

**Issue by Jello146** [Wheellog/Companion.Garmin#78]
If select distance shows travel time.
If select travel time shows travel time.

**Comment by ggoraa** [comment]
yes, issue identified and confirmed as a real issue, the issue will be fixed in 3.0.0 release!

**Comment by ggoraa** [comment]
If you need a version with this fix assp, i can send here a build for your device :)
Just tell me the device version, if you want a build

**Comment by Jello146** [comment]
This is great!
Garmin vivoactive 3
Thanks.

**Comment by ggoraa** [comment]
Sorry for not responding for that long, was very busy
A build will be available tomorrow, trust me :D

**Comment by ggoraa** [comment]
[WheelLog.prg.zip](https://github.com/Wheellog/Companion.Garmin/files/7107154/WheelLog.prg.zip)

**Comment by ggoraa** [comment]
I can't test it rn because of some bluetooth issues on the phone

**Comment by Jello146** [comment]
1. Deleted the application
2. Installed WheelLog.prg file in GARMIN / APPS
3. Runs, but there are no settings in the application Connect IQ and on the watch.

Am I doing something wrong?

**Comment by ggoraa** [comment]
It is intended that you can't access settings of side loaded apps from the phone. But they should be still available on the watch 🤔
